nbs/directives.ipynb

```diff
@@ -0,0 +1,828 @@
+{
```

Comment:
<p>When we put <code>show_docs</code> in backticks here, this is a good opportunity to use a really cool nbdev feature you might not know about that does automated documentation linking. Instead of <code>show_docs</code> put the more qualified name <code>showdoc.show_doc</code> (no "s" at the end, BTW) and it should automatically link to the docs for that!</p><p> </p><p>Update: Jeremy added an entire "explanation" section for show_doc yesterday, which would be good to link to https://nbdev.fast.ai/explanations/showdoc.html</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/nbdev/pull/868/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>